bedrock/firefox/templates/firefox/all-unified.html

```diff
@@ -109,14 +109,19 @@ <h1 class="c-intro-heading">{{ _('Choose which Firefox Browser to download in yo
             </a>
             {{ select_product_list(products, disabled=True) }}
           </p>
-          {# Show the ESR version select dropdown if there are multiple ESR's available. #}
-          {% set hide_esr_version_select = False if desktop_esr_next_version else True %}
 
           {{ product_options('desktop_release', desktop_release_platforms, desktop_release_full_builds, desktop_release_latest_version, disabled=True, hide_version=True, hide_language=False) }}
           {{ product_options('desktop_beta', desktop_beta_platforms, desktop_beta_full_builds, desktop_beta_latest_version, disabled=True, hide_version=True, hide_language=False) }}
           {{ product_options('desktop_developer', desktop_developer_platforms, desktop_developer_full_builds, desktop_developer_latest_version, disabled=True, hide_version=True, hide_language=False) }}
           {{ product_options('desktop_nightly', desktop_nightly_platforms, desktop_nightly_full_builds, desktop_nightly_latest_version, disabled=True, hide_version=True, hide_language=False) }}
+
+          {# Show the ESR version select dropdown if there are multiple ESR's available. #}
+          {% set hide_esr_version_select = False if desktop_esr_next_version else True %}
```

Comment:
style NIT: can be simply 

```suggestion
          {% set hide_esr_version_select = not desktop_esr_next_version %}
```

If you think the original is more readable and understandable then I'm cool with keeping it as is.